src/SwipeButton.tsx

```diff
@@ -28,8 +37,8 @@ interface SwipeButtonProps extends SwipeButtonPropsExtends {
 
   /**
    * Indicates when `onComplete` should be invoked.
-   * 
-   * @default 70
+   *
+   * @default 100
```

Comment:
I'm not sure that changing this default value is the way to go. Is there a reason for changing this to 100% instead of 70% as it was before?

When I developed this package my team and I had a long discussion on why this should not be set to 100%.

Reply:
Yep! Refer to my other comment. I would also be interested to hear the reasoning behind why your needs required a threshold of ~70%.

Reply:
I read your [comment](https://github.com/arelstone/react-native-swipe-button/pull/1#issuecomment-861154282) regarding this. I think we should move the discussion closer to the code, so I will answer here. Hope that is allright with you 

I completely understand that you would like this to be 100%. Finance is something you shouldn't mess with at 70%. 

When developing this component I was using it in an app for artisans/tradesmen to finalize and send an invoice to the customer. Before converting this into a package I had it as a component in the app. At that point we didn't have a complete threshold, meaning that the users would have to swipe to 100%. This resulted in a lot of users reporting that the finalizing part didn't work and that they had troubles finalizing and sending the invoices. Changing this to 70% did the trick. All "bug" reports regarding this stopped and our users was happy. My argument for setting the complete threshold to 70% is that (our type of users) often is in a rush and that if you use your right hand thumb to swipe 70% looks like it is completed. The 70% has been field tested in an app with +20k daily users and it working.

You could argue that it should be 85 or 90% instead. But I think it will make sense to keep this at 70%. As far as I remember the onComplete function will first be invoked when ever the user lifts their finger from the Icon button (I might remember wrong)

Instead of changing the default property I would suggest that you set the `completeThreshold` on your side instead.